docs/build/inferencing.md

```diff
@@ -60,18 +60,23 @@ Basic CPU build
 
 Open Developer Command Prompt for Visual Studio version you are going to use. This will properly setup the environment including paths to your compiler, linker, utilities and header files.
 ```
-.\build.bat --config RelWithDebInfo --build_shared_lib --parallel
+.\build.bat --config RelWithDebInfo --build_shared_lib --parallel --compile_no_warning_as_error --skip_submodule_sync
```

Comment:
Why do we want them to skip module sync? and why do we need this new compile/error flag suddenly?
Also, as someone new to ORT I would like the basic build cmd to include these flags as defaults; I should only need to specify RelWithDebInfo if at all I'm interested in release builds, otherwise just issuing ./build.bat should work.

Reply:
In most cases we do not need the submodules. They are only needed for some special builds. For example, webassembly build needs the emsdk submodule. And nobody would need the onnx submodule except a few build pipelines we have. 

"--compile_no_warning_as_error" is needed when the user's compiler version is different than what we use now. We do not expect the user to help us fix compile warnings, because they are not ONNX Runtime's developers. Now compiler warnings are a main source of build issues reported to us through Github Issues.

Reply:
So, this is temporary and the plan is to resolve this in the next release?